code/modules/antagonists/cult/cult_items.dm

```diff
@@ -344,32 +345,44 @@ Striking a noncultist, however, will tear their flesh."}
 	name = "magus robes"
 	desc = "A set of armored robes worn by the followers of Nar'Sie."
 	icon_state = "magusred"
-	item_state = "magusred"
+	icon = 'icons/obj/clothing/suits/wizard.dmi'
+	worn_icon = 'icons/mob/clothing/suits/wizard.dmi'
+	item_state = null
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS
 	allowed = list(/obj/item/tome, /obj/item/melee/cultblade)
 	armor = list(MELEE = 50,  BULLET = 30, LASER = 50, ENERGY = 20, BOMB = 25, BIO = 10, RAD = 0, FIRE = 10, ACID = 10, STAMINA = 50)
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 
-/obj/item/clothing/head/helmet/space/hardsuit/cult
+/obj/item/clothing/suit/hooded/cultrobes/hardened
```

Comment:
2nd "hardsuit" converted into a hooded suit and given spaceproof. This is because under this pr, typepaths direct towards specific .dmi now.

Also made no sense being a hardsuit